### PR TITLE
WIP feat(server): Add signin authorization endpoint stubs

### LINKED ISSUE #1398: Add support for signin authorization
Details TBD

ref https://docs.google.com/document/d/1qbzfA21T0HoHpgxXlpO26qcCQDx2v0PUbcoHW9XtxQQ

### PR DESCRIPTION
**Pull request by shane-tomlinson**
fixes #1398

### CODE REVIEW

File: lib/routes/account.js [hunk 1]
```diff
@@ -599,6 +599,97 @@ module.exports = function (
...
+            function () {
+              reply({})
+            },
+            reply
+          )
+      }
+    },
+    {
+      method: 'POST',
+      path: '/account/login/reject_authorization_code',
```
Comment (seanmonstar) [inline lib/routes/account.js]: I'm guessing this is a route that the content-server will call if a user sees an email, decides it wasn't them, and clicks "This wasn't me"?

File: lib/customs.js [hunk 2]
```diff
@@ -22,7 +22,12 @@ module.exports = function (log, error) {
   function Customs(url) {
     if (url === 'none') {
       this.pool = {
-        post: function () { return P.resolve({ block: false })},
+        post: function (postUrl, postData) {
+          if (postUrl === '/check' && postData.action === 'accountLogin') {
```
Comment (seanmonstar) [inline lib/customs.js]: These threw me off some. Are these basically supposed to be mocks for running the test suite?

Comment (rfk) [inline lib/customs.js]: FYI this `url === 'none'` branch is used by e.g. self-hosters who don't have customs-server running.

Comment (shane-tomlinson) [inline lib/customs.js]: This is for me to develop the content server w/o having a customs server implementation.

File: lib/routes/account.js (hunk 1 above)
Comment (shane-tomlinson) [inline lib/routes/account.js]: Yeah, the same functionality is going to be added to other verification emails soon too.

**Comment by shane-tomlinson** [comment]
Closing this PR, @seanmonstar has taken over this work.

**Pull request closed without merging**